groups/infrastructure/main.tf

```diff
@@ -0,0 +1,94 @@
+terraform {
+  required_version = "~> 1.3"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.54.0"
+    }
+    vault = {
+      source  = "hashicorp/vault"
+      version = "~> 3.18.0"
+    }
+  }
+}
+
+provider "aws" {
+  region  = var.aws_region
+}
+
+terraform {
+  backend "s3" {}
+}
+
+module "file_transfer_alb" {
+  source = "git@github.com:companieshouse/terraform-modules//aws/application_load_balancer?ref=1.0.296"
+  count = "${var.file_transfer_create_alb == 1 ? 1 : 0}"
+
+  environment         = var.environment
+  service             = "file-transfer"
+  ssl_certificate_arn = data.aws_acm_certificate.cert.arn
+  subnet_ids          = values(local.routing_subnet_ids)
+  vpc_id              = data.aws_vpc.vpc.id
+  idle_timeout        = 1200
+  route53_domain_name     = var.cert_domain
+
+  create_security_group  = true
+  internal               = true
+  ingress_cidrs          = ["0.0.0.0/0"]
+  service_configuration  = {
+    listener_config = {
+        default_action_type = "fixed-response"
+        protocol            = "HTTPS"
+        port                = 443
+      }
+  }
+}
+
+
+module "secure_file_transfer_alb" {
+  source = "git@github.com:companieshouse/terraform-modules//aws/application_load_balancer?ref=1.0.296"
+  count = "${var.file_transfer_create_alb == 1 ? 1 : 0}"
+
+  environment         = var.environment
+  service             = "secure-file-transfer"
+  ssl_certificate_arn = data.aws_acm_certificate.cert.arn
+  subnet_ids          = values(local.routing_subnet_ids)
+  vpc_id              = data.aws_vpc.vpc.id
+  idle_timeout        = 1200
```

Comment:
20 minutes seems an odd choice for the `idle_timeout` when the default is `60` - is this intentional?

Also consider adding a variable for this, defaulted to an appropriate value to allow for future per-environnment overrides if necessary.

Reply:
This is fixed now. Commit is pushed.